src/RichText/RichText.php

```diff
@@ -479,6 +479,8 @@ private static function imageGallery(array $imgs, array $options = []): string
 
         // Unsanitize images urls
         $imgs = array_map(function ($img) {
+            // URL may contain &amp; which is not handled by Sanitizer::decodeHtmlSpecialChars().
+            $img['src'] = str_replace('&amp;', '&#38;', $img['src']);
```

Comment:
GLPI sanitizing process had nerver encode `&` to `&amp;`. If such entites are present, it probably means that content was encoded using the native `htmlentities()` or `htmlspecialchars()` PHP functions, and it was probably done by a plugin.

Trying to fix it here is a bad idea. Almost everytime a "hack" was added to handle a specific encoding issue, side effects required to add corrections in the following versions.

For me, the solution here is:
1. to find what is the source of the issue;
2. to provide a custom script to the customer to fix the data in its database.

Reply:
I noticed that the URL in the IMG tag is the same as the URL on the A tag surrounding it. However, when reaching document.send.php, GET data are different and the inconsistency begins to appear. It looks like the problem comes from photoswipe library.

Reply:
Got it ! 

Content is properly encoded  in DB, bit when the web page is rendered, we enter in RichText::getSafeHtml(). 

```$content = htmLawed($content, $config);``` does bad job here (https://github.com/glpi-project/glpi/blob/10.0/bugfixes/src/RichText/RichText.php#L68)

Reply:
Description of the PR updated : the bug is caused by Htmlawed, which introduces &amp; in the URL. 

@cedric-anne : I think we cannot do better as long as we use with Htmlawed.